plugins/vibin/skills/repo-status/scripts/summarize_context.py

```
@@ -0,0 +1,157 @@
+USAGE = """Usage: summarize_context.py [repo-status.json]
+
+Summarize repo_context.sh --json output as a compact Markdown table.
+Reads stdin when no file is provided.
+"""
+
+
+def load(path: str | None) -> dict:
+    if path:
+        source = Path(path)
+        if not source.exists():
+            raise SystemExit(f"error: JSON file not found: {source}")
```

Ruff (0.15.15)

[warning] Avoid specifying long messages outside the exception class

(TRY003)